app-lifelike/build.gradle

```diff
@@ -33,21 +33,22 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion "0.1.0-dev09"
+        kotlinCompilerVersion "1.3.70-dev-withExperimentalGoogleExtensions-20200424"
+        kotlinCompilerExtensionVersion "0.1.0-dev12"
     }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'androidx.core:core-ktx:1.2.0'
-    implementation 'androidx.ui:ui-framework:0.1.0-dev09'
-    implementation 'androidx.ui:ui-layout:0.1.0-dev09'
-    implementation 'androidx.ui:ui-material:0.1.0-dev09'
-    implementation 'androidx.ui:ui-tooling:0.1.0-dev09'
-    implementation 'androidx.ui:ui-foundation:0.1.0-dev09'
-    implementation 'com.github.zsoltk:compose-router:0.9.0'
+    implementation 'androidx.core:core-ktx:1.3.0'
+    implementation 'androidx.ui:ui-core:0.1.0-dev12'
+    implementation 'androidx.ui:ui-layout:0.1.0-dev12'
+    implementation 'androidx.ui:ui-material:0.1.0-dev12'
+    implementation 'androidx.ui:ui-tooling:0.1.0-dev12'
+    implementation 'androidx.ui:ui-foundation:0.1.0-dev12'
+    implementation project(':router')
```

Comment:
It's fixed on released version for a reason, exactly to make migration easier.

Sample apps demonstrate Router usage, which doesn't change across Compose dev versions. UI is unimportant (as long as it works). So I'd rather have them use outdated but fixed Compose version than to break them / block upgrading the library itself every time there's a new dev version. 

I'd suggest to revert changes of example apps and only update core.

Reply:
Got it. I'll take that out.

Reply:
Reverted the samples